agent/stats/queue.go

```diff
@@ -139,6 +148,13 @@ func (queue *Queue) GetLastStat() *types.StatsJSON {
 	return queue.lastStat
 }
 
+func (queue *Queue) GetLastNetworkStatPerSec() NetworkStatsPerSec {
+	queue.lock.RLock()
+	defer queue.lock.RUnlock()
+
+	return queue.lastNetworkStatPerSec
+}
+
```

Comment:
Instead of calling this function separately, can we have a function that combines lastStat and lastStatsPerSec in queue.go which is called outside of this file to get the stats?

Reply:
I feel keeping it separate will give us flexibility to use both. Since last stat is raw docker stats, I do not think we should mix them up in the get function.